index.html

```diff
@@ -1,33 +1,55 @@
 <!DOCTYPE html>
 
 <html>
-  <head>
-    <title>BusMall</title>
-    <meta charset="UTF-8">
-    <link rel="stylesheet" href="css/reset.css">
-    <link rel="stylesheet" href="css/style.css">
-  </head>
 
-  <body>
-    <header>
-      <h1>BusMall</h1>
+<head>
+  <title>BusMall</title>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
 
-    </header>
+<body>
+  <header>
+    <h1>BusMall</h1>
 
+  </header>
 
-    <main>
 
-    </main>
+  <main>
 
 
-    <footer>
+    <form id="chooseProduct">
+      <fieldset>
+        <legend>Choose a Product</legend>
 
-    </footer>
-    
+        <label for="item1"><img id="firstImage" src="http://via.placeholder.com/200x200" /></label>
```

Comment:
It's clever (in a good way) to use labels around these images to ensure they fill in the correct radio button!